src/responder/kcm/kcmsrv_ccache.h

```diff
@@ -355,4 +355,19 @@ errno_t kcm_ccache_to_sec_input_json(TALLOC_CTX *mem_ctx,
                                      struct cli_creds *client,
                                      struct sss_iobuf **_payload);
 
+/*
+ * sec_key is a concatenation of the ccache's UUID and name
+ * sec_value is the binary representation of ccache.
+ */
+errno_t sec_kv_to_ccache_binary(TALLOC_CTX *mem_ctx,
+                                const char *sec_key,
+                                struct sss_iobuf *sec_value,
+                                struct cli_creds *client,
+                                struct kcm_ccache **_cc);
+
+/* Convert a kcm_ccache to its binary representation. */
+errno_t kcm_ccache_to_sec_input_binary(TALLOC_CTX *mem_ctx,
```

Comment:
Shall `kcm_ccache_to_sec_input_json()` function also lose `client` argument for consistency?

Reply:
I mean, `kcm_ccache_to_sec_input_json()` doesn't use `client` arg anyway, and it would be consistent with `_binary` counterpart to remove it.

But if we plan to remove JSON support anyway, this doesn't matter of course.